docs/src/_plugins/pre-wrapper.js

```diff
@@ -0,0 +1,23 @@
+module.exports = (md) => {
+    const defaultFenceRenderer = md.renderer.rules.fence;
+
+    md.renderer.rules.fence = (...args) => {
+        const [tokens, index] = args;
+
+        if (/^\s*(?:in)?correct(?!\S)/u.test(tokens[index - 1].info)) {
+            return defaultFenceRenderer(...args);
+        }
+
+        return `
+<div class="code-wrapper">
+    ${defaultFenceRenderer(...args)}
+    <button class="copy-btn" id="copyBtn" aria-labelledby="copy-button-label">
```

Comment:
```suggestion
    <button class="copy-btn" aria-labelledby="copy-button-label">
```
Let's avoid giving the same ID to all buttons, as it's not recommended since IDs should be unique.